login-workflow/src/components/BrandedCardContainer/BrandedCardContainer.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { ReactNode } from 'react';
+import Card, { CardProps as CardPropsType } from '@mui/material/Card';
+import { SxProps, Theme, useTheme } from '@mui/material/styles';
+import { Spinner, SpinnerProps } from '../Spinner';
+import defaultBackgroundImage from '../../assets/images/background.svg';
+import { useInjectedUIContext } from '@brightlayer-ui/react-auth-shared';
+import Box, { BoxProps } from '@mui/material/Box';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+import {
+    getBrandedCardContainerUtilityClass,
+    BrandedCardContainerClassKey,
+    BrandedCardContainerClasses,
+} from './BrandedCardContainerClasses';
+import { cx } from '@emotion/css';
+
+export type BrandedCardContainerProps = BoxProps & {
+    loading?: boolean;
+    backgroundImage?: string;
+    cardStyles?: SxProps<Theme>;
+    CardProps?: CardPropsType;
```

Comment:
Why are these still here if you use the 'slotsProps' prop to handle these? Also not sure why the 'CardProps' prop is capitalized or why 'CardPropsType' isn't just 'CardProps'

Reply:
> Also not sure why the 'CardProps' prop is capitalized or why 'CardPropsType' isn't just 'CardProps'

CardPropsType is an alias of CardProps imported from Material UI. We prefer not to have two variables called "CardProps" floating around in the same scope.